client/plots/barchart.js

```diff
@@ -138,7 +135,12 @@ class Barchart {
 					settingsKey: 'unit',
 					options: [
 						{ label: 'Linear', value: 'abs' },
-						{ label: 'Log', value: 'log', getDisplayStyle: plot => (plot.term2 ? 'none' : 'inline-block') },
+						/** Option commented out because:
+						 * 1. The scale for the plot renders incorrectly
+						 * 2. The bars do not appear to be rendering correctly as a result
+						 * TODOs: Investigate the cause of the rendering issue and re-enable the option
+						 */
+						// { label: 'Log', value: 'log', getDisplayStyle: plot => (plot.term2 ? 'none' : 'inline-block') },
```

Comment:
I think we should keep the `log` option, but per my [comment in the associated issue](https://github.com/stjude/proteinpaint/issues/2337#issuecomment-2452061678), do whichever is currently true below
- if the log radio option is clicked, do not show overlay option
- if there is an overlay, do not show log radio input

Reply:
@creilly8 I see that the log is hidden when `plot.term2` is defined. But the overlay input has to also be hidden or disabled when the log scale is currently active.

Reply:
Addressed in https://github.com/stjude/proteinpaint/pull/2340/commits/45baf03ed659be6a654bd742b0a38417071c5de5